README.md

```diff
@@ -1,49 +1,192 @@
 [![Nextflow](https://img.shields.io/badge/nextflow-%E2%89%A523.04.3-brightgreen.svg)](https://www.nextflow.io/)
 
-# `staramr`: nextflow pipeline
+# `staramrnf`: nextflow pipeline
 
-## Introduction
+**staramrnf: nextflow pipeline** is the nextflow adaptation of [staramr](https://github.com/phac-nml/staramr/)
 
-**staramr: nextflow pipeline** is the nextflow adaptation of the [staramr](https://github.com/phac-nml/staramr/)
+> `staramr` (AMR) scans bacterial genome contigs against the [ResFinder][resfinder-db], [PointFinder][pointfinder-db], and [PlasmidFinder][plasmidfinder-db] databases (used by the [ResFinder webservice][resfinder-web] and other webservices offered by the Center for Genomic Epidemiology) and compiles a summary report of detected antimicrobial resistance genes. The `star`in`staramr` indicates that it can handle all of the ResFinder, PointFinder, and PlasmidFinder databases.
 
-> `staramr` (_AMR) scans bacterial genome contigs against the [ResFinder][resfinder-db], [PointFinder][pointfinder-db], and [PlasmidFinder][plasmidfinder-db] databases (used by the [ResFinder webservice][resfinder-web] and other webservices offered by the Center for Genomic Epidemiology) and compiles a summary report of detected antimicrobial resistance genes. The `star|_`in`staramr` indicates that it can handle all of the ResFinder, PointFinder, and PlasmidFinder databases.
+## Table of Contents
+
+- [Usage](#-Usage)
+- [Input](#-Input)
+- [Output](#-Output)
+- [Parameters](#-Parameters)
 
 ## Usage
```

Comment:
Can you provide a link to `docs/usage.md` in this section.

Reply:
52fe7eb